Backpack/Dialog/Classes/BPKDialogIconView.h

```diff
@@ -15,13 +15,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#import "BPKDialogIconDefinition.h"
 #import <UIKit/UIKit.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface BPKDialogIconView : UIView
-@property(nullable, nonatomic, strong) UIColor *iconBackgroundColor;
-@property(nullable, nonatomic, strong) UIImage *iconImage;
+@property(nonatomic, strong) BPKDialogIconDefinition *iconDefinition;
```

Comment:
This should be `nullable`

Reply:
I'm not sure it should... That would mean we allow an icon view to exist without an icon definition.

Think it should be non-null, set in the constructor

Reply:
```swift
let iconImage = BPKDialogIconView() // calls `init`
// this will crash because the type in swift is incorrectly `DialogIconDefinition`
// when it should be `DialogIconDefinition?`
iconImage.iconDefinition.iconImage 
```